docs/src/pages/components/emptyState/usage.mdx

```diff
@@ -0,0 +1,201 @@
+---
+title: Empty state
+description: The empty state indicates the absence of data and helps guide users on expected content and next steps.
+tabs: ['Usage', 'Code']
+showMobile: true
+keywords: ['Menu', 'Select', 'Listbox', 'Context menu']
```

Comment:
Please change this keywords. Keywords should be synonymous of the Components. Also `showMobile` must be `false` here as this component is not available in Mobile Design System

Reply:
done